### ISSUE TITLE
HandleHoveredItemChange triggers multiple time on some UI elements

### ISSUE THREAD

**Issue by ihor-rud** [fmauNeko/MarketBoardPlugin#95]
Hovering item in Supply Missions UI triggers multiple updates by "HandleHoveredItemChange" function. This basically fires new price check and reset loading bar each second.

**Comment by fmauNeko** [comment]
Which UI ? The Grand Company one ? If so I can't seem to reproduce this issue :/

**Comment by ihor-rud** [comment]
yeah, grand company supply missions and a few others. Here is how it looks for me

https://github.com/user-attachments/assets/5d42227b-03fa-424f-8909-ef2977432558
